lib/metrics.py

```diff
@@ -183,7 +184,7 @@ def check_metric(**params):
       params["width"], params["height"],
       params.get("format2", params["format"])
     ) * params["frames"])
-    res = md5(filename = params["decoded"], numbytes = numbytes)
+    res = md5(filename = params.get("decoded", params["ofile"]), numbytes = numbytes)
```

Comment:
The test should set params["decoded"] instead.

Reply:
as above

Reply:
as above